### PR TITLE
Store accessToken/refreshToken/refreshTokenScope/ExpireIn in SFCC

### PR DESCRIPTION
**Pull request by dumega**
https://app.asana.com/0/1204908810772800/1204913455654711/f

### CODE REVIEW

**Review by serenayanbolt — APPROVED** [review]
LGTM!

File: cartridges/int_bolt_embedded_sfra/cartridge/client/default/js/eventListenerRegistration.js [hunk 1]
```diff
@@ -10,6 +10,10 @@ $(document).ready(function () {
     $('.submit-customer').attr('disabled', 'true');
     const isBoltEmbeddedExists = setInterval(async function () {
         const containerToMount = $('#email-guest').parent().get(0);
+        if (typeof containerToMount === 'undefined') {
+            clearInterval(isBoltEmbeddedExists);
+            return;
+        }
```
Comment (serenayanbolt) [inline cartridges/int_bolt_embedded_sfra/cartridge/client/default/js/eventListenerRegistration.js:16]: what is this block used for?

Comment (dumega) [reply cartridges/int_bolt_embedded_sfra/cartridge/client/default/js/eventListenerRegistration.js:16]: @serenayanbolt  Currently the Embedded Accounts framework on checkout page is associated with email-guest input field, but this HTML element is loaded for non-registered customers only, as a result, when login via SSO, there is a JS error "containerToMount is undefined. Then I add this block to fix this error.

Comment (serenayanbolt) [reply cartridges/int_bolt_embedded_sfra/cartridge/client/default/js/eventListenerRegistration.js:16]: ok, I see, thanks for catching this!

File: cartridges/int_bolt_embedded_sfra/cartridge/controllers/Bolt.js [hunk 2]
```diff
@@ -29,6 +31,10 @@ server.get('FetchOAuthToken', server.middleware.https, function (req, res, next)
         // store OAuth token expire time in milliseconds, 1000 -> ONE_SECOND
         session.privacy.boltOAuthTokenExpire = response.result.expires_in * 1000
             + new Date().getTime();
+        var currentBasket = BasketMgr.getCurrentBasket();
```
Comment (justinBolt1987) [inline cartridges/int_bolt_embedded_sfra/cartridge/controllers/Bolt.js:34]: might need to consider the use case that currentBasket does not exists. Can we use `basketMgr. getCurrentOrNewBasket()`

Comment (dumega) [reply cartridges/int_bolt_embedded_sfra/cartridge/controllers/Bolt.js]: Make sense!

**Pull request merged**